fedimint-server/src/config/mod.rs

```diff
@@ -301,11 +301,8 @@ impl ServerConfigConsensus {
                 .map(|(peer, endpoints)| {
                     let url = PeerUrl {
                         name: endpoints.name.clone(),
-                        url: SafeUrl::parse_without_port(&format!(
-                            "iroh://{}.ed25519",
```

Comment:
If this is the offender, then we just need to add `iroh` to `port_or_known_default`.

But that brings the question, should we have `fm-p2p+iroh://` and `fm-p2p+iroh://`?

Also, if the user is using non-default ports, maybe we should put them there.

And another matter is ... why are we panicking on decoding? Possibly this could be used as a DoS vector.